db/questions.yml

```diff
@@ -376,11 +376,21 @@ light_check_trip_switches:
   question: Have you checked the trip switches in the fuse box?
   answers:
     - text: 'Yes'
-      page: replace_lightbulb
+      next: light_check_light_bulb
 
     - text: 'No'
       page: check_fusebox
 
+light_check_light_bulb:
+  question: Have you replaced the light bulb?
+  answers:
+    - text: 'Yes'
+      desc: lightfitting_problem
```

Comment:
What custom text should appear on this page?

Reply:
Content for page as per copy document - row 38 column E (with Louise's revisions)